<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1217</width>
    <height>735</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWebEngineView" name="m_webView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>771</width>
      <height>661</height>
     </rect>
    </property>
    <property name="url">
     <url>
      <string>about:blank</string>
     </url>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_tbInputFile">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>20</y>
      <width>301</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="m_btnInputFile">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>20</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>,,,</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_btnRefresh">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>660</y>
      <width>411</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_btnSave">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>630</y>
      <width>411</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1217</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
